.circleci/config.yml

```diff
@@ -144,3 +164,4 @@ workflows:
       - jruby-9.2-spec
       - jruby-9.2-ascii_spec
       - jruby-9.2-rubocop
+      - code-climate
```

Comment:
What about adding a branch filter here, adding the `COVERAGE` env to the job, and running the `coverage` task directly? That way you don't need to add any logic to the `Rakefile`.

Reply:
Ah, thank you ❤️

I was looking for that kind of feature and didn’t find it. Now knowing that it’s called `filters`, I found the documentation at https://circleci.com/docs/2.0/configuration-reference/#filters-1